package-lock.json

```diff
@@ -14,6 +14,11 @@
       "resolved": "https://registry.npmjs.org/array-matrix/-/array-matrix-1.0.0.tgz",
       "integrity": "sha1-fv9fEhWY+iMqIg+OAL0lKmoiPok="
     },
+    "async": {
+      "version": "0.2.10",
+      "resolved": "https://registry.npmjs.org/async/-/async-0.2.10.tgz",
+      "integrity": "sha1-trvgsGdLnXGXCMo43owjfLUmw9E="
+    },
     "builtin-modules": {
```

Comment:
Can you remove the async library def in package-lock.json, not sure how it got there :/ might just be stale from some WIP.. strange NPM didn't remove it..